charts/spire/charts/spire-agent/templates/configmap.yaml

```diff
@@ -8,7 +8,7 @@ data:
     agent {
       data_dir = "/run/spire"
       log_level = {{ .Values.logLevel | quote }}
-      server_address = "{{ .Release.Name }}-server"
+      server_address = {{ .Values.server.address | default (printf "%s-server" .Release.Name) | quote }}
```

Comment:
this is harder to read then the original.

Reply:
True, but does a lot more then the original. How do we make it more readable?

Reply:
Think it makes sense to move to helpers?

Reply:
This seems fairly clear to me. Use `server.address` if it exists, otherwise use the default generated value.

Reply:
Moved to helpers and merged in a separate pr.